### PR TITLE
Edit and add a couple hundred quotes

### PR DESCRIPTION
**Pull request by srosebattles**
-Verify quotes from several books against original source, notably Curious Incident of the Dog in the Night-Time and Bridget Jones's Diary. AFAIK both of those books were originally published with British spelling/punctuation but the versions I checked out from my library app had been "translated" to American. I generally left the British English for Curious Incident but since the quotes from Bridget Jones seemed to be more poorly transcribed (a few were even at the wrong time), I just matched them to what the edition I had was.
-Add a number of quotes, notably by British detective novelist P.D. James, Minnesota novelist William Kent Krueger, sci-fi novelist Thomas Hoover, and more.
-Proofread and standardized book and author names. Some of these were clearly typos (e.g. Leo Tostoy), some names could have been standardized either way (e.g. Sir Arthur Conan Doyle vs. Arthur Conan Doyle). Am open to standardizing any of those a different way if preferred. But now at least we don't have TS Eliot, T S Eliot, and T.S. Eliot quotes.
-Moved a few imprecise time quotes around to fill in blank spaces
-Removed several duplicate quotes that were missed in previous rounds of editing
-Deleted a quote from a children's book about telling the time. Can put that back if preferred. It just seemed kinda like cheating, and it was a relatively less interesting quote than the other quotes for that time.
-nsfw'd a couple of quotes, including another suicide reference (how many are there???) and the word "bloody", which isn't used as a curse word in American English but is quite a strong one in British English.
-Caught and fixed several typos, although no doubt I have introduced more due to the sheer number of changes.

### CODE REVIEW

File: litclock_annotated.csv [hunk 1]
```diff
@@ -310,7 +331,6 @@
 03:00|3 a.m.|I found myself typing “serial killer” into the Google search box at 3 a.m.|My Sister, The Serial Killer|Oyinkan Braithwaite|unknown
 03:00|three o’clock|About three o’clock the four couples, frightened at their happiness, were sliding down the Russian mountains, a singular edifice which then occupied the heights of Beaujon, and whose undulating line was visible above the trees of the Champs Élysées.|Les Misérables|Victor Hugo|unknown
 03:00|three o’clock|A few moments later, about three o’clock, Courfeyrec chanced to be passing along the Rue Mouffetard in company with Bossuet. The snow had redoubled in violence, and filled the air.|Les Misérables|Victor Hugo|unknown
-03:00|3|My dear Mr. Sherlock Holmes:<br>There has been a bad business during the night at 3, Lauriston Gardens, off the Brixton Road. Our man on the beat saw a light there about two in the morning, and as the house was an empty one, suspected that something was amiss.|A Study in Scarlet|Sir Arthur Conan Doyle|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:313]: "at 3" in this quote refers to an address, not a time

File: litclock_annotated.csv [hunk 2]
```diff
@@ -515,7 +540,6 @@
...
 05:04|four minutes past five|"Oh yes. His clocks were set at one minute past five, four minutes past five and seven minutes past five. That was the combination number of a safe, 515457. The safe was concealed behind a reproduction of the Mona Lisa. Inside the safe," continued Poirot, with distaste, "were the Crown Jewels of the Russian Royal Family."|The Clocks|Agatha Christie|unknown
 05:04|5.04 a.m.|5.04 a.m. on the substandard clock radio. Because why do people always say the day starts now? Really it starts in the middle of the night at a fraction of a second past midnight.|The Accidental|Ali Smith|unknown
-05:04|four minutes past five|"Oh yes. His clocks were set at one minute past five, four minutes past five and seven minutes past five. That was the combination number of a safe, 515457. The safe was concealed behind a reproduction of the Mona Lisa. Inside the safe, continued Poirot, with distaste, "were the Crown Jewels of the Russian Royal Family.""|The Clocks|Agatha Christie|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:518]: duplicate

File: litclock_annotated.csv [hunk 3]
```diff
@@ -859,34 +910,35 @@
...
 09:00|nine o’clock|There was no rain, as Holmes had foretold, and the morning broke bright and cloudless. At nine o’clock Lestrade called for us with the carriage, and we set off for Hatherley Farm and the Boscombe Pool.|The Adventures of Sherlock Holmes|Sir Arthur Conan Doyle|unknown
 09:00|nine|“Don’t give it another thought, old sport.” The familiar expression held no more familiarity than the hand which reassuringly brushed my shoulder. “And don’t forget we’re going up in the hydroplane tomorrow morning, at nine.”|The Great Gatsby|F. Scott Fitzgerald|unknown
-09:00|nine o’clock|At nine o’clock, one morning late in July, Gatsby’s gorgeous car lurched up the rocky drive to my door and gave out a burst of melody from its three-noted horn. It was the first time he had called on me, though I had gone to two of his parties, mounted in his hydroplane, and, at his urgent invitation, made frequent use of his beach.|The Great Gatsby|F. Scott Fitzgerald|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:889]: duplicate

File: litclock_annotated.csv [hunk 4]
```diff
@@ -1097,51 +1163,67 @@
...
+10:31|10.31 a.m.|Inspector Sanderson has told us that he examined the safe at 10.31 a.m. and found two good-sized drops of blood on the bottom.|The Red Thumb Mark|R. Austin Freeman|sfw
+10:32|10:32 AM|Date: Sat, 16 May 2009 10:32 AM<br/>Subject: RE: Meteorite<br/>Dear Alex<br/> I'm very well (thank you for asking), and I'm delighted to hear you're feeling so much better. With regard to your meteorite, this is an extremely generous offer (and a very welcome one!) but I need to be sure that it's definitely what you want.|The Universe versus Alex Woods|Gavin Extence|sfw
 10:32|ten thirty-two|It was ten thirty-two when the bus finally rolled into view.<br>The bus that came was a new type, not like the one I used to take to high school.|Blind Willow, Sleeping Woman|Haruki Murakami|unknown
-10:35|Five-and-twenty to eleven|Five-and-twenty to eleven. A horrible hour - a macabre hour, for it is not only the hour of pleasure ended, it is the hour when pleasure itself has been found wanting.|Rope|Patrick Hamilton|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1101]: moved

File: litclock_annotated.csv [hunk 5]
```diff
@@ -1097,51 +1163,67 @@
...
 11:00|11 o’clock|A scarlet steam engine was waiting next to a platform packed with people. A sign overhead said Hogwarts Express, 11 o’clock.|Harry Potter and the Philosopher’s Stone|J.K. Rowling|unknown
 11:00|eleven|She quit at eleven. Fuck it. She went around the house to the shed, got the spade and shovel, paused, and added a crowbar. She started out of the shed, went back, and took a screwdriver and an adjustable wrench from the toolbox.|The Tommyknockers|Stephen King|nsfw
 11:00|11 a.m.|Baba Diop spent the final thirty-six hours of his life at Fleury-Mérogis. At 11 a.m. pm on the second day, he was stabbed in the throat with a sharpened comb while being led under escort to the governor’s office.|The Nameless Ones|John Connolly|unknown
-11:00|the eleventh hour|An ending, then. November 11, 1991, at eleven o’clock in the morning, the eleventh hour of the eleventh month. It’s a Monday.|The Robber Bride|Margaret Atwood|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1137]: duplicate

File: litclock_annotated.csv [hunk 6]
```diff
@@ -1249,29 +1335,30 @@
...
 11:58|11.58|And when you go down the steps, it's always 11.58 on the morning of September ninth, 1958.|11/22/63|Stephen King|unknown
 11:58|Two minutes before the clock struck noon|Two minutes before the clock struck noon, the savage baron was on the platform to inspect the preparation for the frightful ceremony of mid-day. The block was laid forth-the hideous minister of vengeance, masked and in black, with the flaming glaive in his hand, was ready. The baron tried the edge of the blade with his finger, and asked the dreadful swordsman if his hand was sure? A nod was the reply of the man of blood.|Burlesques|William Makepeace Thackeray|unknown
 11:59|near to twelve|There is a big grandfather clock there, and as the hands drew near to twelve I don't mind confessing I was as nervous as a cat.|The Adventure of Johnnie Waverley: A Hercule Poirot Story|Agatha Christie|unknown
+12:00|noon|When Sister Anna brought him back to Patmos at noon, Gunther shouted to anyone and everyone within earshot, "I can read! I can read!"|Bright Valley of Love|Edna Hong|sfw
```
Comment (srosebattles) [inline litclock_annotated.csv:1346]: This is the most wholesome quote imho

File: litclock_annotated.csv [hunk 7]
```diff
@@ -1406,21 +1503,22 @@
...
 13:17|1.17pm|1.17pm An arrest is made at Bristol Airport and police confirm that they’ve detained a person in connection with the investigation.|What She Knew|Gilly MacMillan|unknown
 13:17|one seventeen|One seventeen and four seconds. That shorter guy’s really got it made, and gets on a scooter, and that taller one, he goes in. One seventeen and forty seconds. That girl there, she’s got a green ribbon in her hair. Too bad that bus just cut her from view.|The Terrorist, He Watches|Wislawa Szymborska|unknown
 13:18|one eighteen|One eighteen exactly. Was she stupid enough to head inside? Or wasn't she? We'll know before long, When the dead are carried out.|The Terrorist, He Watches|Wislawa Szymborska|unknown
-13:20|one twenty|I woke up and glanced at my watch on the table next to my bed. It was one twenty. My heart was beating furiously. I slid off the bed down onto the carpet, sat cross-legged, and took some deep breaths. Then I held my breath, relaxed my shoulders, sat up straight, and tried to focus. I must have swum too much, I decided, or got too much sun.|Blind Willow, Sleeping Woman|Haruki Murakami|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1414]: This was definitely an a.m. quote and a copy of it is still at that time

File: litclock_annotated.csv [hunk 8]
```diff
@@ -1709,22 +1820,23 @@
...
 16:22|4.22pm|Monday, 4.22pm Washington, D.C. Paul Hood took his daily late-afternoon look at the list of names on his computer monitor.|Balance of Power: Op-Center 05|Tom Clancy, Steve Pieczenik, and Jeff Rovin|unknown
 16:23|4.23|They were hurrying west, trying to reach the river before sunset. The warming-related "adjustments" to Earth's orbit had shortened the winter days, so that now, in January, sunset was taking place at 4.23.|A Visit from the Goon Squad|Jennifer Egan|unknown
 16:24|4:24|Mike winked at Ashley and continued with the remaining greetings and hugs and handshakes. The time was 4:24. Six hours to go. The minutes seemed to just melt away.|Teardrop|Travis Thrasher|unknown
-16:25|twenty-five minutes past four|As I dressed I glanced at my watch. It was no wonder that no one was stirring. It was twenty-five minutes past four. I had hardly finished when Holmes returned with the news that the boy was putting in the horse.|The Adventures of Sherlock Holmes|Sir Arthur Conan Doyle|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1718]: This was definitely an a.m. quotes

File: litclock_annotated.csv [hunk 9]
```diff
@@ -1970,42 +2102,52 @@
 19:00|seven o'clock|… in a word, seen always at the same evening hour, isolated from all its possible surroundings, detached and solitary against its shadowy background, the bare minimum of scenery necessary .. to the drama of my undressing, as though all Combray had consisted of but two floors joined by a slender staircase, and as though there had been no time there but seven o'clock at night.|In Search of Lost Time: Swann's Way|Marcel Proust|unknown
 19:00|seven|The town clock struck seven. The echoes of the great chime wandered in the unlit halls of the library. An autumn leaf, very crisp, fell somewhere in the dark. But it was only the page of a book, turning.|Something Wicked This Way Comes|Ray Bradbury|unknown
 19:00|7.00 p.m.|7.00 p.m. Do maths practice|The Curious Incident of the Dog in the Night-Time|Mark Haddon|unknown
-19:00|seven o'clock|By seven o'clock the orchestra has arrived--no thin five-piece affair but a whole pitful of oboes and trombones and saxophones and viols and cornets and piccolos and low and high drums.|The Great Gatsby|F. Scott Fitzgerald|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1973]: duplicate

File: litclock_annotated.csv [hunk 10]
```diff
@@ -187,23 +202,25 @@
 01:38|one-thirty-eight|At one-thirty-eight am suspect left the Drive-In and drove to seven hundred and twenty three North Walnut, to the rear of the residence, and parked the car.|The Narc|William Edmund Butterworth|unknown
 01:40|one-forty am|March twelfth, one-forty am, she leaves a group of drinking buddies to catch a bus home. She never makes it.|Bones to Ashes|Kathy Reichs|unknown
 01:44|sixteen minutes to two|She knew it was the stress, two long days of stress, and she looked at her watch, sixteen minutes to two, and she almost leaped with fright, a shock wave rippling through her body, where had the time gone?|Trackers|Deon Meyer|unknown
+01:45|1:45|In the end I accepted a glass of claret. It seemed both churlish and censorious to let him drink alone. The carriage-clock stuck 1:45 before he roused himself and said: "Sorry, Cousin. Rather drunk. Be glad of your shoulder. To bed, to sleep, perchance to dream."|The Mistletoe Murder|P.D. James|sfw
```
Comment (srosebattles) [inline litclock_annotated.csv:205]: Realizing that I marked this as sfw, but worth bringing up for discussion whether quotes with alcohol should be marked as nsfw? Obviously not a trigger for me at all, but there's a variety of personal or cultural reasons someone might not be comfortable with alcohol references. Could see it going either way. Or possibly figuring out more granular content controls than just sfw/nsfw at some point in the future.

File: litclock_annotated.csv [hunk 11]
```diff
@@ -966,58 +1066,64 @@
...
+09:47|thirteen minutes to ten|For Hunter, who was trained to note times exactly, the final emergency started at thirteen minutes to ten.|Elegy for a Revolutionary|C.J. Driver|sfw
+09:48|nine-forty-eight|"This is the body I first saw at nine-forty-eight on Friday the fifteenth of October at Innocent House, Innocent Walk, Wapping."|Original Sin|P.D. James|sfw
 09:50|9:50|"I should have known," said Maddy. It was typical for the church to be nearly empty at 9:50 and full at 9:59.|Blood Stained|Gayl Siegel|sfw
 09:50|ten to ten|She reaches Radiance at ten to ten, early for once, and lets herself in with her key, and puts on the mauve-and-aqua smock that Shanita designed for them so the customers will know they aren’t customers themselves.|The Robber Bride|Margaret Atwood|unknown
-09:50|Nine o’clock|Nine o’clock young residents wearing leather elbows talk to Acutes for fifty minutes about what they did when they were little boys. The Big Nurse is suspicious of the crew-cut looks of these residents, and that fifty minutes they are on the ward is a tough time for her.|One Flew Over the Cuckoo’s Nest|Ken Kesey|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1000]: moved

File: litclock_annotated.csv [hunk 12]
```diff
@@ -1880,53 +2075,60 @@
...
 18:30|half past six|At five o'clock the two ladies retired to dress, and at half past six Elizabeth was summoned to dinner.|Pride and Prejudice|Jane Austen|unknown
 18:30|six thirty|It is six thirty. Now the dark night and the deafening racket of the crickets again engulf the garden and the veranda, all around the house|Jealousy|Alain Robbe-Grillet|unknown
 18:30|half-past six|To a casual visitor it might have seemed that Mr. Penicuik, who owned the house, had fallen upon evil days; but two of the three gentlemen assembled in the Saloon at half-past six on a wintry evening of late February were in no danger of falling into this error.|Cotillion|Georgette Heyer|unknown
-18:31|a little after half past six|I had been delayed at a case and it was a little after half past six when I found myself at Baker Street once more|The Adventure of The Blue Carbuncle|Arthur Conan Doyle|unknown
-18:32|around half past six|It is around half past six in the evening. Dusk is gathering in the living room, an early dusk due to the fog which has rolled in from the Sound and is like a white curtain drawn down outside the windows.|Long Day's Journey Into Night|Eugene O'Neill|unknown
```
Comment (srosebattles) [inline litclock_annotated.csv:1923]: Still exists at 18:26

File: litclock_annotated.csv (hunk 10 above)
Comment (JohannesNE) [reply litclock_annotated.csv:211]: I think it is good to leave it as sfw (at least for now). But I agree that it is controversial. Generally I think the nsfw tag should be used quite liberally to make to filter out quotes you might not want showing on your work screensaver or in e.g. a public library.

**Review by JohsEnevoldsen — APPROVED** [review]
That is really impressive and a lot of work. Thank you!

**Pull request merged**